assets/js/base/components/cart-checkout/order-summary/order-summary-item.tsx

```diff
@@ -27,10 +26,16 @@ import ProductBackorderBadge from '../product-backorder-badge';
 import ProductImage from '../product-image';
 import ProductLowStockBadge from '../product-low-stock-badge';
 import ProductMetadata from '../product-metadata';
+import { CartItem } from '../../../../types/type-defs/cart';
 
-const productPriceValidation = ( value ) => mustContain( value, '<price/>' );
+const productPriceValidation = ( value: string ): true | Error =>
```

Comment:
This actually seems like `mustContain` was typed wrong, it never _returns_ an error, it throws.

The definition for `mustContain` should be:

```typescript
export const mustContain = (
	value: string,
	requiredValue: string
): true | never
```

and so the return type of `productPriceValidation` should be changed to match. `never` means that the function won't return (because it throws).